packages/cluster/src/commands/seed.js

```diff
@@ -2,20 +2,16 @@ const { reporter } = require('@dhis2/cli-helpers-engine')
 const { initDockerComposeCache, resolveConfiguration } = require('../common')
 const { seed } = require('../db')
 
-const defaults = require('../defaults')
+const run = async function(argv) {
+    const { name, getCache } = argv
```

Comment:
Should this (`name`, particularly) come from `cfg` not directly from `argv`?

Reply:
Hm, `cfg` doesn't actually have the `name` since it is always passed in as an argument, that's why I always grab it from `argv`.

Reply:
I think `cfg` does have it, since internally it does an `Object.assign(..., argv)` but I might be wrong.  This is fine as-is!